### ISSUE TITLE
parse_pgp_key not defined in notification.py

### ISSUE THREAD

**Issue by vjrj** [globaleaks/GlobaLeaks#1796]
Hi, I'm getting this error trying to save a pgp public key:

```
NameError Name not found globally.&#92;&#92;n&#92;&#92;nTraceback (most recent call last):&#92;&#92;n&#92;&#92;n  File &quot;&#47;usr&#47;local&#47;lib&#47;python2.7&#47;dist-packages&#47;twisted&#47;internet&#47;defer.py&quot;, line 1097, in _inlineCallbacks&#92;&#92;n    result = result.throwExceptionIntoGenerator(g)&#92;&#92;n&#92;&#92;n  File &quot;&#47;usr&#47;local&#47;lib&#47;python2.7&#47;dist-packages&#47;twisted&#47;python&#47;failure.py&quot;, line 389, in throwExceptionIntoGenerator&#92;&#92;n    return g.throw(self.type, self.value, self.tb)&#92;&#92;n&#92;&#92;n  File &quot;&#47;usr&#47;lib&#47;python2.7&#47;dist-packages&#47;globaleaks&#47;handlers&#47;admin&#47;notification.py&quot;, line 121, in put&#92;&#92;n&#92;&#92;n  File &quot;&#47;usr&#47;local&#47;lib&#47;python2.7&#47;dist-packages&#47;twisted&#47;python&#47;threadpool.py&quot;, line 191, in _worker&#92;&#92;n    result = context.call(ctx, function, *args, **kwargs)&#92;&#92;n&#92;&#92;n  File &quot;&#47;usr&#47;local&#47;lib&#47;python2.7&#47;dist-packages&#47;twisted&#47;python&#47;context.py&quot;, line 118, in callWithContext&#92;&#92;n    return self.currentContext().callWithContext(ctx, func, *args, **kw)&#92;&#92;n&#92;&#92;n  File &quot;&#47;usr&#47;local&#47;lib&#47;python2.7&#47;dist-packages&#47;twisted&#47;python&#47;context.py&quot;, line 81, in callWithContext&#92;&#92;n    return func(*args,**kw)&#92;&#92;n&#92;&#92;n  File &quot;&#47;usr&#47;lib&#47;python2.7&#47;dist-packages&#47;globaleaks&#47;orm.py&quot;, line 108, in _wrap&#92;&#92;n    result = function(self.store, *args, **kwargs)&#92;&#92;n&#92;&#92;n  File &quot;&#47;usr&#47;lib&#47;python2.7&#47;dist-packages&#47;globaleaks&#47;handlers&#47;admin&#47;notification.py&quot;, line 82, in update_notification&#92;&#92;n&#92;&#92;n  File &quot;&#47;usr&#47;lib&#47;python2.7&#47;dist-packages&#47;globaleaks&#47;handlers&#47;admin&#47;notification.py&quot;, line 31, in parse_pgp_options&#92;&#92;n    if not remove_key and pgp_key_public != &#92;&#39;&#92;&#39;:&#92;&#92;n&#92;&#92;nNameError: global name &#92;&#39;parse_pgp_key&#92;&#39; is not defined&#92;&#92;n&#39;
2016-10-25 09:11:16+0200 [-] Starting factory &lt;txsocksx.client.SOCKS5ClientFactory instance at 0x7f9793d75ef0&gt;
```

I think that show be added `from globaleaks.security import parse_pgp_key` to `globaleaks/handlers/admin/notification.py`.

**Comment by NSkelsey** [comment]
Hi @vjrj, this issue was fixed last night in 89d389fd5. If you update to `v2.64.10`, the error will go away. Thank you for reporting it.

**Comment by NSkelsey** [comment]
@evilaliv3 from coveralls it is possible to see [that this line was never tested](https://coveralls.io/builds/8492050/source?filename=globaleaks%2Fhandlers%2Fadmin%2Fnotification.py#L31), hence no detection of the throw.
